src/main/java/net/hypixel/nerdbot/generator/impl/tooltip/MinecraftTooltipGenerator.java

```diff
@@ -105,7 +105,12 @@ public MinecraftTooltip parseLore(String input, TooltipSettings settings) {
             if (settings.isEmptyLine()) {
                 builder.withEmptyLine();
             }
-            builder.withLines(LineSegment.fromLegacy(rarity.getFormattedDisplay() + " " + settings.getType(), '&'));
+
+            builder.withLines(LineSegment.fromLegacy(
+                rarity.getFormattedDisplay() +
+                    (settings.getType() == null || settings.getType().isEmpty() ? "" : " " + settings.getType()),
+                '&'
+            ));
```

Comment:
Reply to: https://github.com/SkyBlock-Nerds/NerdBot/pull/340/files#r1871826243 (cant add suggestion properly otherwise)
```suggestion

            String formattedType = settings.getType() == null || settings.getType().isEmpty() ? "" : " " + settings.getType()
            builder.withLines(LineSegment.fromLegacy(rarity.getFormattedDisplay() + formattedType, '&'));
```

Like this?

Reply:
Yes